test/fastly/sumologic_logging_test.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../test_helper'
+
+describe Fastly::SumologicLogging do
+  let(:client)     { Fastly.new(api_key: 'notasecrettestkey', user: 'test@example.com', password: 'password') }
+  let(:service_id) { SecureRandom.hex(6) }
+  let(:version)    { 1 }
+  let(:sumo) { Fastly::SumologicLogging.new({ name: 'test_syslog', service_id: service_id, version: 1 }, client) }
```

Comment:
It doesn't look like we're using `sumo` here. Should we remove it? 
```suggestion
```

Reply:
yup, just copied from the syslog test